llarp/util/fs.hpp

```diff
@@ -57,10 +57,10 @@ namespace llarp
     OpenFileStream(fs::path pathname, std::ios::openmode mode)
     {
       if (EnsurePrivateFile(pathname))
-        return {};
+        return std::nullopt;
 
       std::string f = pathname.string();
-      return T{pathname, mode};
+      return T{f, mode};
```

Comment:
Will `return std::make_optional<T>(pathname, mode);` work here?

Reply:
yes it does